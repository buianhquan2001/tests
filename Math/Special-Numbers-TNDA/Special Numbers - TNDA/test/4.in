3
14 8
35 20
58 22
