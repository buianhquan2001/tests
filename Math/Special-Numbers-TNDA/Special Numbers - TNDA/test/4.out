2744
1501850
11319918
